pages/partials/logout.ejs

```diff
@@ -1,6 +1,8 @@
 <% if (typeof authn_user != 'undefined') { %>
    <% if (authn_user.provider == 'shibboleth') { %>
        <a class="btn btn-secondary" href="<%= plainUrlPrefix %>/logout">Log out</a>
+   <% } else if (authn_user.provider == 'google') { %>
```

Comment:
Why this as opposed to `|| authn_user.provider == 'google'` on the conditional above?

Reply:
laziness? :-)

Reply:
fixed now